test/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required( VERSION 3.13.0 )
-project( "coreSNTP unit test"
+project( "coreSNTP tests"
```

Comment:
Should a version of the tests also be set like is being done in [coreMQTT](https://github.com/FreeRTOS/coreMQTT/pull/276/files#diff-33394812ba204689144fd2f80832db83853ba1cb32403edb4e15fe4893e675fdR3), and [coreHTTP](https://github.com/FreeRTOS/coreHTTP/pull/174/files#diff-33394812ba204689144fd2f80832db83853ba1cb32403edb4e15fe4893e675fdR4) and other repos?

Reply:
Tagging a version in the cmake as well.